doc/conf.py

```diff
@@ -153,37 +171,40 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, 'tfs-pandas.tex', u'tfs-pandas Documentation',
-     u'OMC-TEAM', 'manual'),
+    (master_doc, "tfs-pandas.tex", "tfs-pandas Documentation", "pyLHC/OMC-TEAM", "manual"),
 ]
 
-
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
-    (master_doc, 'tfs-pandas', u'tfs-pandas Documentation',
-     [author], 1)
-]
-
+man_pages = [(master_doc, "tfs-pandas", "tfs-pandas Documentation", [author], 1)]
 
 # -- Options for Texinfo output -------------------------------------------
 
 # Grouping the document tree into Texinfo files. List of tuples
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'tfs-pandas', u'tfs-pandas Documentation',
-     author, 'OMC-TEAM', 'One line description of project.',
-     'Miscellaneous'),
+    (
+        master_doc,
+        "tfs-pandas",
+        "tfs-pandas Documentation",
+        author,
+        "pylhc/OMC-TEAM",
+        "One line description of project.",
+        "Miscellaneous",
+    ),
 ]
 
 # -- Autodoc Configuration ---------------------------------------------------
 
 # Add here all modules to be mocked up. When the dependencies are not met
 # at building time. Here used to have PyQT mocked.
-autodoc_mock_imports = ['PyQt5', 'PyQt5.QtGui', 'PyQt5.QtCore', 'PyQt5.QtWidgets',
-                        "matplotlib.backends.backend_qt5agg",
-                        ]
-
+autodoc_mock_imports = [
+    "PyQt5",
+    "PyQt5.QtGui",
+    "PyQt5.QtCore",
+    "PyQt5.QtWidgets",
+    "matplotlib.backends.backend_qt5agg",
+]
```

Comment:
Same here, do we still need to mock imports?

Reply:
Commented out in 61b976d